frontend/src/mocks-server/__mockdata__/tracedetail.ts

```diff
@@ -0,0 +1,2089 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+export const traceDetailResponse = [
```

Comment:
this is a big mock response as I tried but couldn't shorten this, as response struct is interconnect to each other